### ISSUE TITLE
'Failed to load paths: Unable to create process' error

### ISSUE THREAD

**Issue by ravagerWT** [pypa/pipenv#4704]
### Issue description

I am getting a Failed to load paths error on `pipenv install pandas --verbose`.  This error occurs not only on 'pandas', but also on other packages.

I have searched online for 3 hours and tried following methods to fix this issue, but didn't work.
1. reinstall python, pipenv, pip
2. downgrade to python 3.7.6.
3. downgrade to pipenv 2018.11.26 from the [advise](https://github.com/pypa/pipenv/issues/4220#issuecomment-641730023)

Any solution?

### Expected result

No errors.

### Actual result

```
Installing pandas...
Installing package: pandas
Writing supplied requirement line to temporary file: 'pandas'
Installing 'pandas'
Failed to load paths: Unable to create process using 'd:\program files\python\python39\python.exe -c "import sysconfig, distutils.sysconfig, io, json, sys; paths = {u'purelib': u'{0}'.format(distutils.sysconfig.get_python_lib(plat_specific=0)),u'stdlib': u'{0}'.format(sysconfig.get_path('stdlib')),u'platlib': u'{0}'.format(distutils.sysconfig.get_python_lib(plat_specific=1)),u'platstdlib': u'{0}'.format(sysconfig.get_path('platstdlib')),u'include': u'{0}'.format(distutils.sysconfig.get_python_inc(plat_specific=0)),u'platinclude': u'{0}'.format(distutils.sysconfig.get_python_inc(plat_specific=1)),u'scripts': u'{0}'.format(sysconfig.get_path('scripts')),u'py_version_short': u'{0}'.format(distutils.sysconfig.get_python_version()), }; value = u'{0}'.format(json.dumps(paths));fh = io.open('c:/users/user/appdata/local/temp/tmpru1ypz2q.json', 'w'); fh.write(value); fh.close()"'

Output:
[=   ] Installing pandas...Failed to load paths: Unable to create process using 'd:\program files\python\python39\python.exe -c "import sysconfig, distutils.sysconfig, io, json, sys; paths = {u'purelib': u'{0}'.format(distutils.sysconfig.get_python_lib(plat_specific=0)),u'stdlib': u'{0}'.format(sysconfig.get_path('stdlib')),u'platlib': u'{0}'.format(distutils.sysconfig.get_python_lib(plat_specific=1)),u'platstdlib': u'{0}'.format(sysconfig.get_path('platstdlib')),u'include': u'{0}'.format(distutils.sysconfig.get_python_inc(plat_specific=0)),u'platinclude': u'{0}'.format(distutils.sysconfig.get_python_inc(plat_specific=1)),u'scripts': u'{0}'.format(sysconfig.get_path('scripts')),u'py_version_short': u'{0}'.format(distutils.sysconfig.get_python_version()), }; value = u'{0}'.format(json.dumps(paths));fh = io.open('c:/users/user/appdata/local/temp/tmpgzx6mguy.json', 'w'); fh.write(value); fh.close()"'

Output: 
[==  ] Installing pandas...$ ['C:\\Users\\user\\.virtualenvs\\THDataBase-6z5_VtIS\\Scripts\\pip.exe', 'install', '--verbose', '--upgrade', '--exists-action=i', '-r', 'c:\\users\\user\\appdata\\local\\temp\\pipenv-pjqod52y-requirements\\pipenv-yrjfjypp-requirement.txt', '-i', 'https://pypi.org/simple']
Using source directory: 'C:\\Users\\user\\.virtualenvs\\THDataBase-6z5_VtIS\\src'
Error:  An error occurred while installing pandas!
Error text:
Unable to create process using 'd:\program files\python\python39\python.exe "C:\Users\user\.virtualenvs\THDataBase-6z5_VtIS\Scripts\pip.exe" install --verbose --upgrade --exists-action=i -r c:\users\user\appdata\local\temp\pipenv-pjqod52y-requirements\pipenv-yrjfjypp-requirement.txt -i https://pypi.org/simple'


Installation Failed
```

### Steps to replicate

Provide the steps to replicate (which usually at least includes the commands and the Pipfile).

-------------------------------------------------------------------------------

Please run `$ pipenv --support`, and paste the results here. Don't put backticks (`` ` ``) around it! The output already contains Markdown formatting.

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2020.11.15'`

Pipenv location: `'d:\\program files\\python\\python39\\lib\\site-packages\\pipenv'`

Python location: `'d:\\program files\\python\\python39\\python.exe'`

Python installations found:

  - `3.9.5`: `D:\Program Files\Python\Python39\python.exe`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.9.5',
 'os_name': 'nt',
 'platform_machine': 'AMD64',
 'platform_python_implementation': 'CPython',
 'platform_release': '10',
 'platform_system': 'Windows',
 'platform_version': '10.0.19041',
 'python_full_version': '3.9.5',
 'python_version': '3.9',
 'sys_platform': 'win32'}
```

System environment variables:

  - `ALLUSERSPROFILE`
  - `APPDATA`
  - `CHROME_CRASHPAD_PIPE_NAME`
  - `COMMONPROGRAMFILES`
  - `COMMONPROGRAMFILES(X86)`
  - `COMMONPROGRAMW6432`
  - `COMPUTERNAME`
  - `COMSPEC`
  - `DRIVERDATA`
  - `FPS_BROWSER_APP_PROFILE_STRING`
  - `FPS_BROWSER_USER_PROFILE_STRING`
  - `HOMEDRIVE`
  - `HOMEPATH`
  - `LOCALAPPDATA`
  - `LOGONSERVER`
  - `NUMBER_OF_PROCESSORS`
  - `ONEDRIVE`
  - `ONEDRIVECONSUMER`
  - `ORIGINAL_XDG_CURRENT_DESKTOP`
  - `OS`
  - `PATH`
  - `PATHEXT`
  - `PROCESSOR_ARCHITECTURE`
  - `PROCESSOR_IDENTIFIER`
  - `PROCESSOR_LEVEL`
  - `PROCESSOR_REVISION`
  - `PROGRAMDATA`
  - `PROGRAMFILES`
  - `PROGRAMFILES(X86)`
  - `PROGRAMW6432`
  - `PSMODULEPATH`
  - `PUBLIC`
  - `SESSIONNAME`
  - `SYSTEMDRIVE`
  - `SYSTEMROOT`
  - `TEMP`
  - `TMP`
  - `USERDOMAIN`
  - `USERDOMAIN_ROAMINGPROFILE`
  - `USERNAME`
  - `USERPROFILE`
  - `VBOX_MSI_INSTALL_PATH`
  - `WINDIR`
  - `TERM_PROGRAM`
  - `TERM_PROGRAM_VERSION`
  - `LANG`
  - `COLORTERM`
  - `VSCODE_GIT_IPC_HANDLE`
  - `GIT_ASKPASS`
  - `VSCODE_GIT_ASKPASS_NODE`
  - `VSCODE_GIT_ASKPASS_MAIN`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `PYTHONDONTWRITEBYTECODE`
  - `PIP_SHIMS_BASE_MODULE`
  - `PIP_PYTHON_PATH`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`

Pipenv??pecific environment variables:

Debug??pecific environment variables:

  - `PATH`: `c:\program files (x86)\intel\icls client\;c:\programdata\oracle\java\javapath;c:\program files\intel\icls client\;c:\windows\system32;c:\windows;c:\windows\system32\wbem;c:\windows\system32\windowspowershell\v1.0\;c:\program files (x86)\intel\intel(r) management engine components\dal;c:\program files\intel\intel(r) management engine components\dal;c:\program files (x86)\intel\intel(r) management engine components\ipt;c:\program files\intel\intel(r) management engine components\ipt;c:\program files\crucial\crucial storage executive;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;D:\Program Files\Git\cmd;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;D:\Program Files\Python\Python39\Scripts\;D:\Program Files\Python\Python39\;D:\Program Files\Python\Scripts\;D:\Program Files\Python\;D:\Program Files\Python\Python37\Scripts\;D:\Program Files\Python\Python37\;C:\Users\user\AppData\Local\Microsoft\WindowsApps;;D:\Program Files\Fiddler`
  - `LANG`: `zh_TW.UTF-8`

---------------------------

Contents of `Pipfile` ('D:\\Coding\\TH\\THDataBase\\Pipfile'):

```toml
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
pandas = "*"

[dev-packages]

```

Contents of `Pipfile.lock` ('D:\\Coding\\TH\\THDataBase\\Pipfile.lock'):

```json
{
    "_meta": {
        "hash": {
            "sha256": "ebffa69a1fa192d1cef7cb42ad79231ca976565c5ce371a70160b3048d3cbc06"
        },
        "pipfile-spec": 6,
        "requires": {},
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {},
    "develop": {}
}

```
</details>

**Comment by licheegh** [comment]
I install python into c:\python37\, then this issue fixed.
path contains space or something need "".

**Comment by matteius** [comment]
@ravagerWT Can this be closed or have you tried latest `2022.1.8` or the main branch?

**Comment by ravagerWT** [comment]
I go back using pipenv for a while, so I don't know if this issue being resolved or not.  But I think it could be closed if you wish.
